kong/runloop/plugin_servers/mp_rpc.lua

```diff
@@ -329,7 +329,7 @@ function Rpc:handle_event(plugin_name, conf, phase)
   local _, err = bridge_loop(self, instance_id, phase)
 
   if err then
-    kong.log.err(err)
+    kong.log.info(err)
 
     if string.match(err:lower(), "no plugin instance") then
```

Comment:
The original `err` could be a "real" error; I prefer to change the log level to WARN inside the `if` condition.

```suggestion
    if string.match(err:lower(), "no plugin instance") then
      kong.log.warn(err)
```

Then continue to print an error level msg if it won't match.